dev-docs/publisher-api-reference/bidderSettings.md

```diff
@@ -45,8 +45,8 @@ Some sample scenarios where publishers may wish to alter the default settings:
 | suppressEmptyKeys | standard or adapter-specific | 0.13.0 | false | If custom adserverTargeting functions are specified that may generate empty keys, this can be used to suppress them. |
 | allowZeroCpmBids | standard or adapter-specific | 6.2.0 | false | Would allow bids with a 0 CPM to be accepted by Prebid.js and could be passed to the ad server. |
 | storageAllowed | standard or adapter-specific | 6.13.0 | true | Allow use of cookies and local storage. |  
-| allowAlternateBidderCodes | standard or adapter-specific | 6.23.0 | true in v6.x <br /> false from v7.0| Allow adapters to bid with alternate bidder codes. |  
-| allowedAlternateBidderCodes | standard or adapter-specific | 6.23.0 | n/a | Array of bidder names for which an adapter can bid. <br />`undefined` or `['*']` will allow adapter to bid with any bidder code. |  
+| allowAlternateBidderCodes | standard or adapter-specific | 6.23.0 | false | Allow adapters to bid with alternate bidder codes. |  
```

Comment:
I'm thinking that the message "true in v6.x <br /> false from v7.0" is still useful here. People will be on 6.x for many months and will read this doc.